SQL/schema.sql

```diff
@@ -0,0 +1,32 @@
+/**
+* Specifies the schema for the project
+*/
+
+DROP TABLE IF EXISTS company_account CASCADE;
+DROP TABLE IF EXISTS router CASCADE;
+DROP TABLE IF EXISTS cve CASCADE;
+
+CREATE TABLE company_account (
+	company_name VARCHAR(64) NOT NULL,
+	email VARCHAR(355) PRIMARY KEY,
+	password VARCHAR(50) NOT NULL,
+	contact CHAR(8) NOT NULL,
+	address VARCHAR(355) NOT NULL,
+	postal_code CHAR(6) NOT NULL
+);
+
+CREATE TABLE router (
+	company_email VARCHAR(355) REFERENCES company_account(email) ON UPDATE CASCADE ON DELETE CASCADE,
+	mac_address CHAR(17) PRIMARY KEY,
```

Comment:
Interestingly, it seems like `CHAR` does not check the length automatically and will just allow the insertion anyway if the input is too short etc. Seems like we might have to implement the check ourselves before the insertion.